docs/public-networks/concepts/transactions/pool.md

```diff
@@ -90,4 +95,28 @@ or free gas networks:
 * If you set [`zeroBaseFee`](../../reference/genesis-items.md) to `true` in the genesis file,
   the transaction pool price bump is set to `0`. Specifying a value for transaction pool price bump using [`--tx-pool-price-bump`](../../reference/cli/options.md#tx-pool-price-bump)
   will cause an error.
-* If the [minimum gas price is zero](../../../private-networks/how-to/configure/free-gas.md), the transaction pool price bump is set to `0`, unless you specify a different value using [`--tx-pool-price-bump`](../../reference/cli/options.md#tx-pool-price-bump). 
\ No newline at end of file
+* If the [minimum gas price is zero](../../../private-networks/how-to/configure/free-gas.md), the transaction pool price bump is set to `0`, unless you specify a different value using [`--tx-pool-price-bump`](../../reference/cli/options.md#tx-pool-price-bump).
+
+## Transaction pool options and methods
+
+Options and methods for configuring and monitoring the transaction pool include the following:
+
+|                                                                                              | Description                                                                      | Transaction pool type|
+|----------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------|----------------------|
+| [`txpool_besuTransactions`](../../reference/api/index.md#txpool_besutransactions)            | API method to list transactions in the transaction pool.                         | Both                 |
```

Comment:
There are two more `txpool_` API methods to add to this table: https://besu.hyperledger.org/public-networks/reference/api#txpool-methods